RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.h

```diff
@@ -226,11 +232,11 @@ class CAHitNtupletGeneratorKernels {
 
   TupleMultiplicity const* tupleMultiplicity() const { return device_tupleMultiplicity_.get(); }
 
-  void launchKernels(HitsOnCPU const& hh, TkSoA* tuples_d, cudaStream_t cudaStream);
+  void launchKernels(HitsConstView hh, TkSoAView track_view, cudaStream_t cudaStream);
 
-  void classifyTuples(HitsOnCPU const& hh, TkSoA* tuples_d, cudaStream_t cudaStream);
+  void classifyTuples(HitsConstView hh, TkSoAView track_view, cudaStream_t cudaStream);
 
-  void buildDoublets(HitsOnCPU const& hh, cudaStream_t stream);
+  void buildDoublets(HitsConstView hh, int32_t offsetBPIX2, cudaStream_t stream);
```

Comment:
I'm confused why `offsetBPIX2` now is passed as an additional parameter, and why in oher places it is actually part of the SoA - but we can leave the explanation for later ...